api/base/settings/defaults.py

```diff
@@ -73,8 +73,8 @@
     # Needs to go before CommonMiddleware, so that transactions are always started,
     # even in the event of a redirect. CommonMiddleware may cause other middlewares'
     # process_request to be skipped, e.g. when a trailing slash is omitted
+    'api.base.middleware.FlaskRequestMiddleWare',
     'api.base.middleware.TokuTransactionsMiddleware',
-
```

Comment:
Like in our import spec, this blank line seems to serve as separation between middlewares that are built into Django, and those we define. As order is important I'm not sure this separation will always be possible, but I'd like to keep the blank line until it isn't.